src/polyswarm_api/api.py

```diff
@@ -234,7 +229,7 @@ def submit(self, artifact, artifact_type=resources.ArtifactType.FILE, artifact_n
         if hasattr(artifact, 'read') and hasattr(artifact.read, '__call__'):
             artifact = resources.LocalArtifact.from_handle(self, artifact, artifact_name=artifact_name or '',
                                                            artifact_type=artifact_type)
-        elif isinstance(artifact, string_types):
+        elif isinstance(artifact, str):
```

Comment:
That in fact changes the behavior from accepting `bytes | unicode` to `unicode` only. Hope it to have no impact.

Reply:
Yes I took a careful look to this because Python strings is a thing, so I went straight to the source code of the "future" library, and they [say](https://github.com/PythonCharmers/python-future/blob/2bdbfa5664241fee622b23590abd3492f973afc9/src/future/utils/__init__.py#L21) this about `string_types`:

> string_types: basestring in Python 2, str in Python 3

And if you go to the actual [source code](https://github.com/PythonCharmers/python-future/blob/2bdbfa5664241fee622b23590abd3492f973afc9/src/future/utils/__init__.py#L161) that defines the type for `PY3`, indeed it's just a reference to the `str` type:

> `string_types = str,`